db/entity/DiscordMember.ts

```diff
@@ -25,10 +25,13 @@ export class DiscordMember {
   discordMemberId: string;
 
   @Column()
-  discordServerId: string;
+  DiscordServerId: string;
```

Comment:
please rename `discordServerId `, not sure why you added the uppercase?

```suggestion
  discordServerId: string;
```